eng/nativepgo.targets

```diff
@@ -60,8 +60,4 @@
 
     <Error Condition="!Exists('$(PgoPackagePath)') And '$(NativeOptimizationDataSupported)' == 'true'" Text="Unable to locate restored PGO package at $(PgoPackagePath). Maybe the platform-specific package naming changed?" />
   </Target>
-
```

Comment:
nit: on line 36 the comment says `DumpPgoDataPackagePath` but the target was renamed to `GetPgoDataPackagePath`.